.github/workflows/snapshots.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020-2021 Dave Verwer, Sven A. Schmidt, and other contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Snapshots
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  snapshots:
+    name: Image Snapshots
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Build Front-End
+        run: make build-front-end
+      - name: Copy Front-End
+        run: cp -a ./Public/* ./Tests/AppTests/__Snapshots__/WebpageSnapshotTests
+      - name: Exclude NoImageSnapshot Tests
+        run: rm ./Tests/AppTests/__Snapshots__/WebpageSnapshotTests/*_noImageSnapshots*.html
+      - id: diff-includes-html-files
+        name: Diff for HTML Changes
+        run: |
+          filecount=$(git diff origin/main --shortstat **.html | awk '{print $1}')
+          echo "::set-output name=matching-files::${filecount:-0}"
```

Comment:
As discussed, here's my take if you like it better:

```
bash-3.2$ git diff --name-only origin/main | grep .html$ | wc -l | bc
29
bash-3.2$ filecount=$(git diff origin/main --shortstat **.html | awk '{print $1}')
bash-3.2$ echo ${filecount:-0}
29
```

The `${...:-0}` is to convert the empty string when there's not diff to `0`.

Reply:
For reference:

```
bash-3.2$ git diff origin/main --shortstat **.html
 29 files changed, 7361 insertions(+)
```

Reply:
Much better without the external file, too! 👍